src/data/list/alist.lean

```diff
@@ -198,6 +198,10 @@ theorem insert_entries_of_neg {a} {b : β a} {s : alist β} (h : a ∉ s) :
   (insert a b s).entries = ⟨a, b⟩ :: s.entries :=
 by rw [insert_entries, kerase_of_not_mem_keys h]
 
+theorem insert_of_neg {a} {b : β a} {s : alist β} (h : a ∉ s) :
```

Comment:
Similar lemmas are called `of_not_mem` here and there.

Reply:
I'm matching the name of `alist.insert_entries_of_neg`. I can rename these two lemmas at a later time.

Reply:
Please add a TODO for this, so it doesn't get forgotten